src/DefaultBuilder/test/Microsoft.AspNetCore.TrimmingTests/SlimBuilderDoesNotDependOnX509Test.cs

```diff
@@ -0,0 +1,10 @@
+using Microsoft.AspNetCore.Builder;
+
+var builder = WebApplication.CreateSlimBuilder(args);
+var app = builder.Build();
+
+if (X509Utilities.HasCertificateType) {
+    return 1;
```

Comment:
(nit) The convention is to use negative numbers. That way if you ever get more than 99 checks, you don't have to skip over 100. And it is easier to intuit that a negative exit code means something went wrong.

```suggestion
    return -1;
```

Reply:
Ah, the classic hundred-failure-modes unit test...